test/AsmResolver.PE.Tests/DotNet/Metadata/MetadataTest.cs

```diff
@@ -198,5 +203,51 @@ public void SelectFirstUserStringsStreamInEnCMetadata()
         {
             AssertCorrectStreamIsSelected<UserStringsStream>(Properties.Resources.HelloWorld_DoubleUserStringsStream_EnC, true);
         }
+
+        [Fact]
+        public void SchemaStreamShouldForceEnCMetadata()
+        {
+            var peImage = PEImage.FromBytes(Properties.Resources.HelloWorld_SchemaStream);
+            AssertCorrectStreamIsSelected<BlobStream>(peImage, true);
+            AssertCorrectStreamIsSelected<GuidStream>(peImage, true);
+            AssertCorrectStreamIsSelected<StringsStream>(peImage, true);
+            AssertCorrectStreamIsSelected<UserStringsStream>(peImage, true);
+        }
+
+        [Fact]
+        public void UseCaseInsensitiveComparisonForHeapNamesInEnCMetadata()
```

Comment:
We probably want a counterpart test that tests whether case sensitivity is used for non-enc metadata (e.g., two binaries with both a `#Strings` and a `#strings` stream, one EnC and one non-EnC metadata).